sdk/eventhub/azure-eventhub-checkpointstoretable/azure/eventhub/extensions/checkpointstoretable/_tablestoragecs.py

```diff
@@ -17,22 +56,299 @@ class TableCheckpointStore:
+                  The format is like "<namespace>.servicebus.windows.net".
+                - `eventhub_name` (str): The name of the specific Event Hub the checkpoint is associated with,
+                  relative to the Event Hubs namespace that contains it.
+                - `consumer_group` (str): The name of the consumer group the checkpoint is associated with.
+                - `partition_id` (str): The partition ID which the checkpoint is created for.
+                - `sequence_number` (int): The sequence number of the :class:`EventData<azure.eventhub.EventData>`
+                  the new checkpoint will be associated with.
+                - `offset` (str): The offset of the :class:`EventData<azure.eventhub.EventData>`
+                  the new checkpoint will be associated with.
+        :rtype: None
+        """
+        try:
+            checkpoint_entity = TableCheckpointStore._create_checkpoint_entity(
+                checkpoint
+            )
+            self._table_client.update_entity(
+                mode=UpdateMode.REPLACE, entity=checkpoint_entity, **kwargs
+            )
+        except ResourceNotFoundError:
+            logger.info(
+                "Create checkpoint entity %r because it hasn't existed in the table yet.",
+                checkpoint_entity,
+            )
+            self._table_client.create_entity(entity=checkpoint_entity, **kwargs)
+
+    def claim_ownership(self, ownership_list, **kwargs):
+        # type: (Iterable[Dict[str, Any]], Any) -> Iterable[Dict[str, Any]]
+        """Tries to claim ownership for a list of specified partitions.
+
+        :param Iterable[Dict[str,Any]] ownership_list: Iterable of dictionaries containing all the ownerships to claim.
+        :rtype: Iterable[Dict[str,Any]], Iterable of dictionaries containing partition ownership information:
+                - `fully_qualified_namespace` (str): The fully qualified namespace that the Event Hub belongs to.
+                  The format is like "<namespace>.servicebus.windows.net".
+                - `eventhub_name` (str): The name of the specific Event Hub the checkpoint is associated with,
+                  relative to the Event Hubs namespace that contains it.
+                - `consumer_group` (str): The name of the consumer group the ownership are associated with.
+                - `partition_id` (str): The partition ID which the checkpoint is created for.
+                - `owner_id` (str): A UUID representing the owner attempting to claim this partition.
+                - `last_modified_time` (float): The last time this ownership was claimed.
+                - `etag` (str): The Etag value for the last time this ownership was modified. Optional depending
+                  on storage implementation.
+        """
+        gathered_results = []
+        for x in ownership_list:
+            gathered_results.append(self._claim_one_partition(x), **kwargs)
```

Comment:
I think this was a typo:
```suggestion
            gathered_results.append(self._claim_one_partition(x, **kwargs))
```